fist/server.c

```diff
@@ -44,7 +56,70 @@ struct connection_info
     dstring last_command;
 };
 
-static int do_delete(hashmap *hm, int fd, dstringa params) {
+static int do_help(struct config *config, hashmap *hm, int fd, dstringa params) {
+    send(fd, HELP, strlen(HELP), 0);
+    return 0;
+}
+
+static int do_keys(struct config *config, hashmap *hm, int fd, dstringa params) {
```

Comment:
Should the keys be formatted as a JSON array, like the output of search?

Reply:
Ah, yes it should. That is a really good point.